common/src/main/res/values/styles.xml

```diff
@@ -287,8 +291,8 @@
     </style>
 
     <style name="EncircledIconTheme">
-        <item name="backgroundColor">@color/steel_gray_40</item>
-        <item name="strokeColor">@color/steel_gray_40</item>
+        <item name="backgroundColor">@color/ultra_light_gray</item>
+        <item name="strokeColor">@color/ultra_light_gray</item>
```

Comment:
https://github.com/dashevo/dash-wallet/blame/4518027cefc965dafd271ede4bf42f12e2d8b1cb/common/src/main/res/values/styles.xml#L294-L295

This change was made a month ago, probably as part of a PR on this branch.

Reply:
Yeah `steel_gray_` and `gray_` colors are obsolete

Reply:
ok, great!